doc/source/tutorial/stats/continuous_studentized_range.rst

```diff
@@ -0,0 +1,27 @@
+
+.. _continuous-studentized_range:
+
+Studentized Range Distribution
+==============================
+This distribution has two shape parameters :math:`\nu>0` and :math:`k>0` and support is :math:`q\geq0`.
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+        f(q;k,\nu) = \frac{\nu^{\nu/2}}{\Gamma(\nu/2)\,2^{\nu/2-1}}\,  \int_0^\infty\int_{-\infty}^\infty  s^{\nu-1}\,e^{-\nu s^{2}/2}  \,sk(k-1) \varphi(z) \varphi(sq+z) [\Phi(sq+z)-\Phi(z)]^{k-2} \,\mathrm{d}z \, \mathrm{d}s\\
```

Comment:
The equations are taken from the Wikipedia. I've checked, and it seems like if you flip the endpoints of both integrals in a double integral, the sign change cancels out. This could just be a different convention.

Regarding the _k(k-1)_ term, I'll move it outside the integral - it's not in there for any particular reason other than that's how I wrote it, and it's probably more clear moved outside.